specs/multicloud/Huawei_Cloud_File_Block.md

```diff
@@ -0,0 +1,931 @@
+        "KmsKeyId": "<KmsKeyId>",
+        "KmsKeyName": "<KmsKeyName>"
+    },
+    "metadata": {
+        "fields": {
+            "CreatedAt": {
+                "Kind": {
+                    "StringValue": "2020-10-15T02:53:47.914879Z"
+                }
+            },
+            "ExportLocation": {
+                "Kind": {
+                    "StringValue": "<ExportLocation>"
+                }
+            },
+            "ExportLocations": {
+                "Kind": {
+                    "StringValue": "<ExportLocation>"
+                }
+            },
+            "Host": {
+                "Kind": {
+                    "StringValue": ""
+                }
+            },
+            "HwSFSType": {
+                "Kind": {
+                    "StringValue": "SFS"
+                }
+            },
+            "IsPublic": {
+                "Kind": {
+                    "BoolValue": false
+                }
+            },
+            "Links": {
+                "Kind": {
+                    "StructValue": {
+                        "fields": {
+                            "href": {
+                                "Kind": {
+                                    "StringValue": "<Links>"
+                                }
+                            },
+                            "rel": {
+                                "Kind": {
+                                    "StringValue": "self"
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "ProjectID": {
+                "Kind": {
+                    "StringValue": "<ProjectID>"
+                }
+            },
+            "ShareID": {
+                "Kind": {
+                    "StringValue": "cf149d37-09f6-45eb-8c73-8713f0abffeb"
+                }
+            },
+            "ShareNetworkID": {
+                "Kind": {
+                    "StringValue": ""
+                }
+            },
+            "ShareType": {
+                "Kind": {
+                    "StringValue": "default"
+                }
+            },
+            "SnapshotID": {
+                "Kind": {
+                    "StringValue": ""
+                }
+            },
+            "VolumeType": {
+                "Kind": {
+                    "StringValue": "default"
+                }
+            }
+        }
+    }
+}
+```
+**Delete File share**
+
+DELETE http://192.168.20.162:8089/v1/94b280022d0c4401bcf3b0ea85870519/file/shares/5f87b9bc90f7e10001fa260c
+
+Response:
+
+```
+{}
+```
+
+## Dependencies
+
+The support for Huawei cloud SFS and EVS will use the GOLANG SDK provided. Here is the reference:
+
+### SFS
+[https://github.com/huaweicloud/golangsdk](https://github.com/huaweicloud/golangsdk)
+
+### EVS
+[https://github.com/huaweicloud/huaweicloud-sdk-go-v3](https://github.com/huaweicloud/huaweicloud-sdk-go-v3)
+
+## Dashboard changes: 
```

Comment:
do we have details here ? specifically 

1. any extra filters added in the listing etc. for file and block ?
2. what is the dashboard behaviour for services like versioning, encryption, migration etc. when the level of support is varied across source and destination ?
3. how is uneven service support handled across backend types (across object, file and block) on the dashboard and internal API ?

Reply:
I will update this part. Thought this can come as separate spec from Dashboard